pkg/db/transactions.go

```diff
@@ -163,7 +163,7 @@ func withTransactionAllOptions[T any](h *DBHandler, ctx context.Context, opts tr
 
 func (h *DBHandler) BeginTransaction(ctx context.Context, readonly bool) (*sql.Tx, error) {
 	return h.DB.BeginTx(ctx, &sql.TxOptions{
-		Isolation: sql.LevelRepeatableRead,
+		Isolation: sql.LevelReadCommitted,
```

Comment:
I think this is ok, but I would appreciate a comment (in the code) explaining how we got here.
E.g. which situation (parallel create-releases?) leads to an error with repeatable read?

Reply:
Done